app/main.py

```diff
@@ -1,3 +1,37 @@
+from dataclasses import dataclass
+
+
+@dataclass
 class Distance:
-    # Write your code here
-    pass
+    km: float
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: "Distance | float") -> "Distance":
```

Comment:
Import annotation from `future` to use this types without quotes. other can also be `int` her. Check all other cases